### PR TITLE
Move newly added functions to `mywxtools.h`

### PR DESCRIPTION
**Pull request by t2b3**
Similiar functions are stored in `mywxtools.h`, so better to keep it that way.

### CODE REVIEW

File: src/mywxtools.h [hunk 1]
```diff
@@ -179,3 +179,14 @@ static wxBitmap ConvertBufferToWxBitmap(vector<uint8_t> &buf, wxBitmapType bmt)
     wxBitmap bm(im, 32);
     return bm;
 }
+
+inline static bool IsWordSep(wxChar ch) {
```
Comment (aardappel) [inline src/mywxtools.h:183]: Are these used in multiple files? If they are only used in the selection code, it be best to move them close to where they are used

Comment (t2b3) [reply src/mywxtools.h]: Hi :-) Thanks for your feedback. It looks like the callers are only in `selection.h` itself, so I updated the PR/commit.

**Comment by t2b3** [comment]
@aardappel Thanks for the merge. Have a nice day!

**Comment by AntonBogun** [comment]
ahh I didn't mention but yes those functions will end up being used again in text.h since I'm planning to fix how selection of a word with a mouse happens...

**Pull request merged**